torch_tvm/register.cpp

```diff
@@ -53,6 +55,11 @@ PYBIND11_MODULE(_torch_tvm, m) {
       FuseConcat(g);
       RemoveDropout(g);
       FuseSupportedOps(g);
+      if (debug) {
```

Comment:
Can this get into `DebugLogger` too?

Reply:
Let me think. Might be a bit tricky, but would be good to put alltogether.